src/index.js

```diff
@@ -2,18 +2,25 @@ const isBrowser = typeof require === 'undefined'
 const fetch = isBrowser ? window.fetch : require('node-fetch')
 
 class PublicGoogleSheetsParser {
-  constructor (spreadsheetId) {
+  constructor (spreadsheetId, tabName) {
```

Comment:
Could you rename from tabName to sheetName?

according to below:
https://developers.google.com/sheets/api/guides/concepts

They don't use 'tab'.

Reply:
Good point lol, I think I got the word tab from the stackoverflow question 😆